cmd/nodecmd/create.go

```diff
@@ -57,6 +57,7 @@ var (
 	cmdLineGCPCredentialsPath       string
 	cmdLineGCPProjectName           string
 	cmdLineAlternativeKeyPairName   string
+	devnetNumRPCNodes               int
```

Comment:
can we call this devnetNumAPINodes? and replace rpc with api where appropriate

Reply:
sure